hls4ml/model/optimizer/passes/bn_fuse.py

```diff
@@ -1,38 +1,97 @@
+import numpy as np
+
 from hls4ml.model.layers import BatchNormalization, Conv1D, Conv2D, Dense
 from hls4ml.model.optimizer import OptimizerPass
+from hls4ml.model.types import FixedPrecisionType, IntegerPrecisionType, UnspecifiedPrecisionType
 
 
 class FuseBatchNormalization(OptimizerPass):
+    """
+    OptimizerPass to merge BatchNormalization layers,
+    only if the earlier one does not have quantization specified
+
+    Note:  Consider restricting this to ApplyAlpha.  Batch Normalization quantization seems to be ignored.
+
+    Note:  This optimizer may not be safe if weights are updateable. May need to turn off.
+    """
```

Comment:
This seems to be the doc string from ` FuseConsecutiveBatchNormalization` above and should probably be replaced with one appropriate to this function.

Reply:
Attempted to update.